constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -231,6 +245,72 @@ export class ClassEditPage extends React.Component {
                         onChange={(e) => this.handleChange(e, "inputClassKey")}
                     />
 
+                    <InputText
+                        label="Google Classroom Code"
+                        isTextBox={true}
+                        required={false}
+                        description="Enter the google classroom code"
+                        value={this.state.googleClassCode}
+                        onChangeCallback={(e) =>
+                            this.handleChange(e, "googleClassCode")
+                        }
+                        validators={[
+                            {
+                                validate: (text) => text != "",
+                                message: "You must input a description",
+                            },
```

Comment:
This field can be optional so this field is allowed to be empty. I think we don't need a validator here since the GoogleCode can be a mixture of alphanumerical characters.